client/src/planwise/client/projects2/components/settings.cljs

```diff
@@ -224,26 +235,33 @@
   [read-only build-actions upgrade-actions]
   (let [current-project (subscribe [:projects2/current-project])
         build-actions   (subscribe [:projects2/build-actions])
-        upgrade-actions (subscribe [:projects2/upgrade-actions])]
+        upgrade-actions (subscribe [:projects2/upgrade-actions])
+        analysis-type   (get-in @current-project [:config :analysis-type])]
     [:section {:class-name "project-settings-section"}
      [section-header 5 "Actions"]
      [:div {:class "step-info"} "Potential actions to increase access to services. Planwise will use these to explore and recommend the best alternatives."]
-     [project-setting-title "account_balance" "Available budget"]
-     [current-project-input "" [:config :actions :budget] "number" "$" "" {:disabled read-only :class "project-setting"}]
-     [m/TextFieldHelperText {:persistent true} "Planwise will keep explored scenarios below this maximum budget"]
 
-     [project-setting-title "domain" "Building a new provider..."]
-     [listing-actions {:read-only?  read-only
-                       :action-name :build
-                       :list        @build-actions}]
+     [project-setting-title "info" "Budget"]
+     [current-project-checkbox "Do you want to analyze scenarios using a budget?" [:config :analysis-type] "budget" "action" {:disabled read-only :class "project-setting"}]
+
+     (when (not= analysis-type "action")
```

Comment:
Why is `(not= analysis-type "action")` instead of `(= analysis-type "budget")` used here?

Reply:
Same here, I'll update and check that it works.